dev-qt/qtwebengine/qtwebengine-5.15.14_p20240510.ebuild

```diff
@@ -81,6 +80,7 @@ RDEPEND="
 	pulseaudio? ( media-libs/libpulse )
 	screencast? ( media-video/pipewire:= )
 	system-icu? ( >=dev-libs/icu-69.1:= )
+	system-re2? ( <=dev-libs/re2-0.2022.12.01:= )
```

Comment:
Let's do this unconditionally, please. qtwebengine:5 is on life support and the flag will have to go away soon enough anyway when old re2 is cleaned up.

It's ok if you keep the patch but just then hardwire the configure arg.

Reply:
Done in 0aa41db